discovery/db_discovery.go

```diff
@@ -212,3 +221,21 @@ func pmTicketParams(params *net.TicketParams) *pm.TicketParams {
 		Seed:              new(big.Int).SetBytes(params.Seed),
 	}
 }
+
+func (dbo *DBOrchestratorPoolCache) PollOrchestratorInfo(bcast common.Broadcaster) {
+	ticker := getTicker()
+	go func() {
+		for range ticker.C {
```

Comment:
At the moment, this loop will never exit and thus the goroutine will never exit gracefully. One fix could be to pass in a `context.Context` into `PollOrchestratorInfo()` and then exit the loop when a `select` receives a value on the context's `Done()` channel. The caller of this method could create the context with a cancel function which is called on program exit.

Reply:
fixed in 9458ef19